gnomad/resources/grch38/reference_data.py

```diff
@@ -19,6 +19,55 @@
 from gnomad.utils.vep import vep_or_lookup_vep
 
 
+def _import_gtex_rsem(gtex_path: str, meta_path: str, **kwargs) -> hl.MatrixTable:
```

Comment:
Is this exactly the same as the one in grch37? If so, we should probably just import it from there

Reply:
It is, I just wonder if you used this function to get that MT, it seems the same structure as I see in the MT.

Reply:
also because it's giving a warning this underscore named function: access to a protected member * of a module

Reply:
Yeah, don't worry about the warning, or remove the `_`. Let me confirm, but I think it's the same function

Reply:
I did this 
```
from gnomad.resources.grch37.reference_data import _import_gtex_rsem

try:
    mt = hl.read_matrix_table("gs://gnomad/resources/grch38/gtex/gtex_rsem_v10.mt")
except:
    mt = _import_gtex_rsem(
        gtex_path="gs://gnomad/resources/gtex/v10/GTEx_Analysis_2022-06-06_v10_RSEMv1.3.3_transcripts_tpm.txt.bgz",
        meta_path="gs://gnomad/resources/gtex/v10/GTEx_Analysis_v10_Open_Access_Reduced_Annotations_SampleAttributesDS.txt.bgz",
        min_partitions=1000,
    
    ).checkpoint("gs://gnomad/resources/grch38/gtex/gtex_rsem_v10.mt", _read_if_exists=True)
```